CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 4.14.0
+
+- Allow use of multiple GA customDimensionIndex.
```

Comment:
You added some great documentation for the two new Analytics features, how about adding a link to them from the change log note? (so something updating can find the new options easily)